manager/src/main/resources/db/migration/0.1/V0.1.11__update_bridge.sql

```diff
@@ -0,0 +1,11 @@
+-- add new "organisation_id" column
+alter table BRIDGE
+    add column "organisation_id" varchar(255);
+
+-- update existing BRIDGE "organisation_id" with '15247674' value (the only possible one before this migration)
+update BRIDGE
+set "organisation_id"='15247674';
```

Comment:
@vaibhavjainwiz How have you determined this organisation id?

Reply:
I had taken its value from `event-bridge-fm-realm-sample.json`
https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/c2273a08b77d342e9ecc11fcfcc74d57fcb75e61/kustomize/overlays/minikube/keycloak/event-bridge-fm-realm-sample.json#L119

Reply:
This value is hardcoded only in local environment, both dev and stable has this dynamic (actually every account recognised by sso.redhat.com).

Please remove this migration part, we cannot hardcore the value for all the environments.

@robpblake 
This is an interesting use case: proper migration for this change requires probably some custom code that does sso queries and update the value. It is not a use case we ever implemented (afaik).

Reply:
@danielezonca Correct, we've never stored the `org_id` before. 

I agree with your description of the proper migration, however I'd suggest that we do a full environment reset when we deploy this PR. I don't see there being any value in maintaining a DB migration that applies the correct organisation_id to existing `Bridges`, particularly in our `dev` environment. 

We know that at some point soon we're going to ask NS to migrate to the app-interface hosted Fleet Manager, so that's another reason as well to not worry too much about it.

cheers,

Rob

Reply:
@robpblake @danielezonca 
I had removed default value initialization from migration script. As discussed now we have to reset every env.